src/triage/component/postmodeling/contrast/model_evaluator.py

```diff
@@ -222,7 +222,7 @@ def feature_group_importances(self, path=None):
 			SELECT
 			model_id,
 			feature_group,
-			avg(feature_importance) as importance_average
+			max(feature_importance) as importance_average
```

Comment:
maybe rename to `importance_aggregate` or `importance_max` to avoid confusion? How much does that column name get used downstream?